test/imp/max-inconsistent-prelude-spec.k.out.golden

```diff
@@ -1,6 +1,6 @@
-kore-exec: [233424] Error (ErrorException):
```

Comment:
This test must have been silently failing for a long time...

Reply:
The test actually only searches the output for a particular string with `grep`. I will update the test in another pull request so that this is clearer.